tools/Requirements-Checker/checker.php

```diff
@@ -51,19 +51,24 @@
 	'message' => ini_get('memory_limit'),
 );
 
-$tests['hf'] = array(
-	'title' => '.htaccess file protection',
-	'required' => FALSE,
-	'description' => 'File protection by <code>.htaccess</code> is not present. You must be careful to put files into document_root folder.',
-	'script' => '<script src="assets/denied/checker.js"></script> <script>displayResult("hf", typeof fileProtectionChecker == "undefined")</script>',
-);
-
-$tests['hr'] = array(
-	'title' => '.htaccess mod_rewrite',
-	'required' => FALSE,
-	'description' => 'Mod_rewrite is probably not present. You will not be able to use Cool URL.',
-	'script' => '<script src="assets/rewrite/checker"></script> <script>displayResult("hr", typeof modRewriteChecker == "boolean")</script>',
-);
+$software = (isset($_SERVER['SERVER_SOFTWARE']) ? strtolower($_SERVER['SERVER_SOFTWARE']) : 'unknown');
+if ($software === 'unknown' || strpos($software, 'apache') !== FALSE) {
```

Comment:
Why not

~~~php


if (!isset($_SERVER['SERVER_SOFTWARE']) || stripos($_SERVER['SERVER_SOFTWARE'], 'Apache') !== FALSE) {